pydicom/pixel_data_handlers/util.py

```diff
@@ -21,33 +21,45 @@ def convert_color_space(arr, current, desired):
         or (rows, columns, planes).
     current : str
         The current color space, should be a valid value for (0028,0004)
-        *Photometric Interpretation*. One of ``'RGB'``, ``'YBR_FULL'``.
-        For ``YBR_FULL_422`` use ``'YBR_FULL'``.
+        *Photometric Interpretation*. One of ``'RGB'``, ``'YBR_FULL'``,
+        ``'YBR_FULL_422'``.
     desired : str
         The desired color space, should be a valid value for (0028,0004)
-        *Photometric Interpretation*. One of ``'RGB'``, ``'YBR_FULL'``.
-        For ``YBR_FULL_422`` use ``'YBR_FULL'``.
+        *Photometric Interpretation*. One of ``'RGB'``, ``'YBR_FULL'``,
+        ``'YBR_FULL_422'``.
 
     Returns
     -------
     numpy.ndarray
         The image(s) converted to the desired color space.
-    """
-    if not HAVE_NP:
-        raise ImportError(
-            "Numpy is required to convert the color space."
-        )
```

Comment:
I removed this because its redundant (can't convert a numpy array if you don't already have numpy)